### PR TITLE
updated the default metrics for the datasets

### PR DESCRIPTION
**Pull request by SwordSaintLancelot**
## Why are you creating this Pull Request?
The default metric has been added to all the layers of every dataset for the US GHG center.

### CODE REVIEW

**Review by j08lue — CHANGES_REQUESTED** [review]
@SwordSaintLancelot there seem to be indentation errors that caused the preview build to fail. See the CI job logs for details, please.

**Comment by SwordSaintLancelot** [comment]
@j08lue I did look into the logs and it mentioned that the issue is with duplicate mapping key. I checked the indentation and it seems to be correct (at least similar to the example that you did)

File: datasets/noaa-cpfp-co2-point.data.mdx [hunk 1]
```diff
@@ -70,7 +68,10 @@ layers:
         - '#f89540'
         - '#fdac33'
         - '#fdc527'
-        - '#f8df25'    
+        - '#f8df25'  
+    analysis:
```
Comment (slesaad) [inline datasets/noaa-cpfp-co2-point.data.mdx:72]: can you add back `exclude: true` in the analysis?

File: datasets/emit-ch4plume-v1.data.mdx [hunk 2]
```diff
@@ -42,7 +42,8 @@ layers:
     description: Methane point source plume complex extents with methane enhancements
     initialDatetime: newest
     analysis:
-      exclude: true
+      metrics:
```
Comment (slesaad) [inline datasets/emit-ch4plume-v1.data.mdx:45]: can you add back `exclude: true` here

Comment (slesaad) [reply datasets/emit-ch4plume-v1.data.mdx:45]: same for all the epa layers

**Review by slesaad — CHANGES_REQUESTED** [review]
Also don't push .veda/ui changes?

**Comment by slesaad** [comment]
@SwordSaintLancelot don't add exclude to all the datasets, only to those that already had it

**Comment by SwordSaintLancelot** [comment]
> Also don't push .veda/ui changes?

Reverted them to their original state.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "veda-config",
   "description": "Configuration for Veda",
-  "version": "1.0.0-beta.2",
+  "version": "0.1.0",
```
Comment (slesaad) [inline package.json:4]: 🤔

Comment (j08lue) [reply package.json:4]: Looks like this branch here should be rebased to current `develop` or something? Where you did you branch off from, @SwordSaintLancelot?

Comment (SwordSaintLancelot) [reply package.json:4]: I did branch it from develop. I will rebase it and push again.

Comment (j08lue) [reply package.json:4]: Thanks. I was just worried about the "reverting" commits above. If these changes are already in `develop` by now _and mean to be there_, your commits here would undo them.

Comment (SwordSaintLancelot) [reply package.json:4]: I think there were a few changes that happened when I ran the yarn serve and that's what modified the /.veda/ui folder along with package.json but I changed those to original package.json which was in the develop branch.

**Review by slesaad — APPROVED** [review]

**Pull request merged**